compiler/packages/babel-plugin-react-compiler/src/ReactiveScopes/CodegenReactiveFunction.ts

```diff
@@ -2328,6 +2328,7 @@ function codegenJsxAttribute(
 }
 
 const JSX_TEXT_CHILD_REQUIRES_EXPR_CONTAINER_PATTERN = /[<>&]/;
+const JSX_TEXT_BRACKET_REQUIRES_EXPR_CONTAINER_PATTERN = /[{}]/;
```

Comment:
Nit: instead of a new regex, can you add brackets to the existing one?

Reply:
sure